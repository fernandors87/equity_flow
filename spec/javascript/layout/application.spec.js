import Application from 'layout/application'
import Chrono from 'models/chrono'
import { List } from 'immutable'
import { Provider } from 'react-redux'
import configureMockStore from 'redux-mock-store'
import thunk from 'redux-thunk'

const states = {
  accounts: List(),
  chrono: new Chrono('2017-09-11', '2018-01-07'),
  splits: List(),
}

const middlewares = [thunk]
const mockStore = configureMockStore(middlewares)
const store = mockStore(states)

it('should render the component', () => {
  const wrapper = mount(
    <Provider store={ store }>
      <Application />
    </Provider>
  )

  expect(wrapper.html()).toMatchSnapshot()
})
