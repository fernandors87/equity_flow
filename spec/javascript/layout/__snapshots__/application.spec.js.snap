// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the component 1`] = `"<div class=\\"application\\"><div class=\\"navbar-top\\"><nav class=\\"navbar navbar-default\\"><div class=\\"container-fluid\\"><div class=\\"navbar-header\\"><a href=\\"#home\\" class=\\"navbar-brand\\">EquityFlow</a></div></div></nav></div><div class=\\"content\\"><div class=\\"home\\"><table class=\\"monthly-expenses table table-hover\\"><thead><tr><th>Account</th><th>2017-09</th><th>2017-10</th><th>2017-11</th><th>2017-12</th><th>2018-01</th><th>Sum</th><th>Avg</th></tr></thead><tbody></tbody><tfoot><tr><td class=\\"category\\">Total</td><td class=\\"value\\">0</td><td class=\\"value\\">0</td><td class=\\"value\\">0</td><td class=\\"value\\">0</td><td class=\\"value\\">0</td><td class=\\"summary total\\">0</td><td class=\\"summary avg\\">0</td></tr></tfoot></table></div></div></div>"`;
